CONTRIBUTING.md

```diff
@@ -32,23 +32,27 @@ No responsibility for any financial or other damage.
 
 ## What can I do right now?
 
-* Doing code reviews
-* Writing automated tests to increase code coverage
+* Do code reviews
+* Write automated tests to increase code coverage
```

Comment:
Here, @SunyataZero , he corrected the verb tenses to match the heading `What can I do right now?`

Actually, `Do` doesn't need to be there. "What can I do?" answer... "Code reviews".. The "do" is implied.

Reply:
Should I change that @andy5995 ?